.github/workflows/python-publish.yml

```diff
@@ -66,4 +66,4 @@ jobs:
       env:
         DISCORD_WEBHOOK: ${{ secrets.DISCORD_WEBHOOK }}
       with:
-        args: 'Launchable CLI {{ EVENT_PAYLOAD.release.tag_name }} is released! {{ EVENT_PAYLOAD.release.html_url }}'
```

Comment:
Can't we use `needs.tagpr.outputs.tag`?

Only 'Launchable CLI is released!` is no meaning...

e.g.) current message format
![image](https://github.com/launchableinc/cli/assets/536667/4ecac378-7523-4667-add4-2813186c7f71)

Reply:
I checked `needs.tagpr.outputs.tag` works 👍 

Ref:
https://github.com/ono-max/vscode-launchable/blob/main/.github/workflows/release.yaml#L34
https://github.com/ono-max/vscode-launchable/actions/runs/5794230906/job/15703421203#step:4:5